Calibration/HcalCalibAlgos/test/RecAnalyzerHF.cc

```diff
@@ -241,13 +243,13 @@ void RecAnalyzerHF::endJob() {
 	cells++;
       }
     }
-    edm::LogInfo("AnalyzerHF") << "cells" << " " << cells;
+    edm::LogVerbatim("AnalyzerHF") << "cells" << " " << cells;
 #ifdef EDM_ML_DEBUG
     std::cout << "cells" << " " << cells << std::endl;
 #endif
   }
 #ifdef EDM_ML_DEBUG
-  std::cout << "Exiting from RecAnalyzerHF::endjob" << std::endl;
+  edm::LogVerbatim("AnalyzerHF") << "Exiting from RecAnalyzerHF::endjob";
```

Comment:
In the other statements wrapped by `EDM_ML_DEBUG` you use `std::cout`. I suggest to choose one way. Using the message logger is clearly nicer, but since it is anyway hidden by `EDM_ML_DEBUG`, I have no strong opinion here.